internal/command/commands/env.go

```diff
@@ -31,7 +31,7 @@ func (e *Env) Run(args []string) int {
 	env := flagSet.String("env-type", "dev", "environment to attach with environment")
 	service := flagSet.String("service", "", "service name to filter out describe environment")
 	component := flagSet.String("component", "", "component name to filter out describe environment")
-	providerAccount := flagSet.String("account", "", "account name to provision the environment in")
+	providerAccount := flagSet.String("account", "staging", "account name to provision the environment in")
```

Comment:
can we set defaults from backend ?

Reply:
Yes Suraj that sounds better, closing this pr